src/services/expenseService.js

```diff
@@ -0,0 +1,52 @@
+let expenses = [];
+let expenseId = 1;
+
+const resetExpenses = () => {
+  expenses = [];
+};
+
+const getAllExpense = () => {
```

Comment:
Consider renaming `getAllExpense` to `getAllExpenses` to better reflect that it returns a list of expenses.